inputfiles/overridingTypes.jsonc

```diff
@@ -140,13 +140,13 @@
                 }
             },
             "HTMLOrSVGElement": {
-              "properties": {
-                "property": {
-                  "nonce": {
-                    "name": "nonce?"
-                  }
+                "properties": {
```

Comment:
I think we'll once need to format this file correctly, but for now can you revert the formatting so that the PR can have a single purpose?